sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -70,8 +70,53 @@ APIs for using IotHub v2 jobs
 </details>
 
 <details><summary><b>Messages</b></summary>
-Feedback messages, sending cloud to device messages (missing from current swagger), and purging cloud to device message queue
+Ssending cloud to device messages, receiving feedback messages, and purging cloud to device message queue
+
 ```csharp
+public class CloudToDeviceMessages
+{
+    /// <summary>
+    /// Send cloud to device message for a device.
+    /// </summary>
+    /// <param name="deviceId">The unique identifier of the device.</param>
+    /// <param name="message">The cloud to device message to be sent to the device. </param>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns>The Http response.</returns>
+    public virtual async Task<Response> SendMessageAsync(String deviceId, CloudToDeviceMessage message, CancellationToken cancellationToken = default) { }
+
+    /// <summary>
+    /// Retrieve feedback notification for cloud to device messages.
+    /// </summary>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns>The message feedback batch, containing the details of the final state of sent messages.</returns>
+    public virtual async Task<Response<MessageFeedbackBatch>> GetMessageFeedbackAsync(CancellationToken cancellationToken = default) { }
```

Comment:
It was brought up in the design meeting that these method names can be unclear, any suggestions on what to rename them to?

Reply:
This one looks fine, we were discussing about purge one afaik